params/avalanche_params.go

```diff
@@ -23,13 +23,13 @@ const (
 	CortinaGasLimit       uint64 = 15_000_000
 
 	ApricotPhase3MinBaseFee               int64  = 75 * GWei
-	ApricotPhase3MaxBaseFee               int64  = 225 * GWei
+	ApricotPhase3MaxBaseFee                      = 225 * GWei
 	ApricotPhase3InitialBaseFee           int64  = 225 * GWei
-	ApricotPhase3TargetGas                uint64 = 10_000_000
+	ApricotPhase3TargetGas                       = 10_000_000
 	ApricotPhase4MinBaseFee               int64  = 25 * GWei
 	ApricotPhase4MaxBaseFee               int64  = 1_000 * GWei
-	ApricotPhase4BaseFeeChangeDenominator uint64 = 12
-	ApricotPhase5TargetGas                uint64 = 15_000_000
+	ApricotPhase4BaseFeeChangeDenominator        = 12
+	ApricotPhase5TargetGas                       = 15_000_000
```

Comment:
By allowing these to be untyped ints the compiler can just interpret the type as it is needed.